app/services/webhook_manager/donation_creator.rb

```diff
@@ -19,9 +19,13 @@ def call
                                                   seller_id: seller_id,
                                                   payment_intent: payment_intent
                                                 })
+        amount_after_fees = WebhookManager::FeeHandler.call({
+          payment_intent: payment_intent,
+          amount: amount
+        })
```

Comment:
@Jeremy-Xue just wanted to double check this - since .call is an instance method, you might have to initialize with `.new(payment_intent:, amount:)` first?